gui/wxpython/startup/utils.py

```diff
@@ -55,6 +58,54 @@ def get_possible_database_path():
     return path
 
 
+def create_possible_database_path():
+    """Create the directory to what is possibly a GRASS Database.
+
+    Create directory named grassdata in the usual locations.   
+    
+    Returns the new path as a string.  
+    """
+    home = os.path.expanduser('~') 
+    tmp = os.path.join(tempfile.gettempdir(),
+                               "grassdata_{}".format(getpass.getuser()))           
+    
+    # Create independent "grassdata" in the home Linux dir
+    if sys.platform.startswith('linux'):
+        try:
+            path = os.path.join(home, "grassdata")
+            os.mkdir(path)
+        except OSError: 
+            print ("Creation of the directory {} failed".format(path))
```

Comment:
We may decide to refine it and give more info to the user through GUI, but at this point and in this function, there is no need to report the failure in any way. This function has a fallback which is the tmp dir and the original calling code has its own fallback solution.